queries.js

```diff
@@ -0,0 +1,96 @@
+const promise = require('bluebird')
+
+const options = {
+  promiseLib: promise
+}
+
+const pgp = require('pg-promise')(options)
+const connectionString = 'postgres://localhost:5432/possiblegazerbookstore'
+const db = pgp(connectionString)
+
+function getAllBooks(request, response, next) {
+  db.any('SELECT * FROM books')
+    .then(function(data) {
+      response.status(200)
+        .json({
+          status: 'success',
```

Comment:
I typically avoid the use of `status` and `message` keys in an API response - the HTTP status code should provide this information.  (The only time I might use a `message` is if the server is intended to respond in a specific language for the purpose of localization and internationalization.)